app/Jobs/RockTheVote/ImportRockTheVoteRecord.php

```diff
@@ -226,17 +226,19 @@ public static function shouldUpdateStatus($currentStatus, $newStatus)
     /**
      * Update Northstar user with record data.
      *
-     * @param NorthstarUser $user
+     * @return void
      */
-    private function updateUserIfChanged($user)
+    public function updateUserIfChanged(NorthstarUser $user)
```

Comment:
As a future refactor, I still think we could get rid of the conditionals and and move that logic into respective called methods, but not a big deal for now.